docs/integrations/ai-engines/mindsdb_inference.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: MindsDB Inference Endpoints
+sidebarTitle: MindsDB Inference Endpoints
+---
+
+This documentation describes the integration of MindsDB with [MindsDB Inference Endpoints](https://mindsdb-docs.hashnode.space/), a cloud service that simplifies the way developers interact with cutting-edge LLMs through a universal API.
+
+## Prerequisites
+
+Before proceeding, ensure the following prerequisites are met:
+
+1. Install MindsDB [locally via Docker](https://docs.mindsdb.com/setup/self-hosted/docker) or use [MindsDB Cloud](https://cloud.mindsdb.com/).
+2. To use MindsDB Inference Endpoints within MindsDB, install the required dependencies following [this instruction](/setup/self-hosted/docker#install-dependencies).
+3. Obtain the MindsDB Inference API key required to deploy and use MindsDB Inference Endpoints models within MindsDB. Follow the [instructions for obtaining the API key](https://mindsdb-docs.hashnode.space/docs/authentication).
+
+## Setup
+
+Create an AI engine from the [MindsDB Inference Endpoints handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/mindsdb_inference_handler).
+
+```sql
+CREATE ML_ENGINE mindsdb_serve
+FROM mindsdb_inference
+USING
+      mindsdb_inference_api_key = 'api-key-value'
+```
+
+Create a model using `mindsdb_serve` as an engine.
+
+```sql
+CREATE MODEL mindsdb_inference_model
+PREDICT answer
+USING
+      engine = 'mindsdb_serve',   -- engine name as created via CREATE ML_ENGINE
+      model_name = 'model-name',              -- choose one of available models
+      prompt_teplate = 'prompt-to-the-model'; -- prompt message to be completed by the model
+      mode = 'mode_name' -- choose one of the available modes: 'conversational', 'conversational-full', 'image', 'embedding'
```

Comment:
Based on the list of supported models, I believe `image` and `embedding` are not supported, right? If so, we should update both docs files.